dvclive/catalyst.py

```diff
@@ -0,0 +1,28 @@
+from catalyst.core.callback import Callback, CallbackOrder
+
+import dvclive
+
+
+class DvcLiveCallback(Callback):
+    def __init__(self, model_file=None):
+        super().__init__(order=CallbackOrder.external)
+        self.model_file = model_file
+
+    def on_epoch_end(self, runner) -> None:
+        step = runner.stage_epoch_step
+
+        for loader_key, per_loader_metrics in runner.epoch_metrics.items():
+            for key, value in per_loader_metrics.items():
+                key = key.replace("/", "_")
+                key += "_" + loader_key
+                dvclive.log(key, float(value), step)
```

Comment:
I think it would be nice to have `loader_key` ("train/" and "valid/" ) as prefix as some other frameworks also use this schema.

The `/` replace in `key` still needs to happen due to key duplication.

```suggestion
                key = key.replace("/", "_")
                dvclive.log(f"{loader_key}/{key}", float(value), step)
```

Reply:
Yes, you're right, it's a good suggestion